backend/clubs/models.py

```diff
@@ -244,6 +246,20 @@ def __str__(self):
     def create_thumbnail(self, request=None):
         return create_thumbnail_helper(self, request, 200)
 
+    def add_ics_events(self):
+        url = self.ics_import_url
+        if url is not None:
```

Comment:
To be more robust, you can do `if url:` so that an empty string also does nothing instead of throwing an error.